### ISSUE TITLE
Pull in expat security fixes

### ISSUE THREAD

**Issue by jgm** [the-real-blackh/hexpat#2]
I don't know if there has been an expat release yet, but the problem has been fixed in their repository and Debian has patched:

```
- -------------------------------------------------------------------------
Debian Security Advisory DSA-3582-1                   security@debian.org
https://www.debian.org/security/                     Salvatore Bonaccorso
May 18, 2016                          https://www.debian.org/security/faq
- -------------------------------------------------------------------------

Package        : expat
CVE ID         : CVE-2016-0718

Gustavo Grieco discovered that Expat, an XML parsing C library, does not
properly handle certain kinds of malformed input documents, resulting in
buffer overflows during processing and error reporting. A remote
attacker can take advantage of this flaw to cause an application using
the Expat library to crash, or potentially, to execute arbitrary code
with the privileges of the user running the application.

For the stable distribution (jessie), this problem has been fixed in
version 2.1.0-6+deb8u2. Additionally this update refreshes the fix for
CVE-2015-1283 to avoid relying on undefined behavior.

We recommend that you upgrade your expat packages.

Further information about Debian Security Advisories, how to apply
these updates to your system and frequently asked questions can be
found at: https://www.debian.org/security/
```

**Comment by the-real-blackh** [comment]
Thanks!

**Comment by hartwork** [comment]
There is a regression fix related to the vulnerability mention above that you may want as well, see
https://sourceforge.net/p/expat/code_git/ci/af507cef2c93cb8d40062a0abe43a4f4e9158fb2/

**Comment by hartwork** [comment]
There is Expat 2.2.1 by now with more security fixes. Please update.
https://github.com/libexpat/libexpat/blob/master/expat/Changes

**Comment by the-real-blackh** [comment]
@hartwork, thank you for poking me. I'll do this soon.

**Comment by the-real-blackh** [comment]
Done and released on hackage as hexpat-0.20.11.

**Comment by hartwork** [comment]
Awesome, thank you!
